src/components/SearchClustersPage.vue

```diff
@@ -139,12 +152,13 @@ export default {
     },
     handlePaginationPageChanged(page) {
       this.$navigation.updateQueryParameters({
-        [QueryParameters.PageNumber]: page - 1
+        [QueryParameters.PageNumber]: page,
```

Comment:
So `-1` has been removed to make the page query parameter `1`-based?

Reply:
yep, to keep in sync with what you see in your URL (see the watcher on $route.query)